### PR TITLE
fix:修复bug 折扣为负数的问题 并且设置了折扣的取值范围

### PR DESCRIPTION
**Pull request by RuoChengF**
针对 issues 中 
https://github.com/gocrane/crane/issues/754
https://github.com/gocrane/crane/issues/757
bug 的修复 

详细描述:
 折扣这里 会存在负数的问题

### CODE REVIEW

File: pkg/web/src/modules/editClusterSlice.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/*
+ * @Description: 
+ * @Author: 若城
+ * @Date: 2023-05-04 22:20:52
+ * @LastEditTime: 2023-05-04 22:23:49
+ */
```
Comment (qmhu) [inline pkg/web/src/modules/editClusterSlice.ts:6]: Please remove this

File: pkg/web/src/modules/editClusterSlice.ts [hunk 2]
```diff
@@ -22,7 +28,7 @@ const defaultCluster = {
   id: v4(),
   clusterName: '',
   craneUrl: '',
-  discount: 100,
+  discount: 1,
```
Comment (qmhu) [inline pkg/web/src/modules/editClusterSlice.ts:31]: Why set the discount to 1? default value is 100.

File: pkg/web/src/modules/editClusterSlice.ts (hunk 1 above)
Comment (RuoChengF) [reply pkg/web/src/modules/editClusterSlice.ts:6]: 好的

File: pkg/web/src/modules/editClusterSlice.ts (hunk 2 above)
Comment (RuoChengF) [reply pkg/web/src/modules/editClusterSlice.ts:31]: 折扣的取值范围是 1-9.9

File: pkg/web/src/modules/editClusterSlice.ts (hunk 1 above)
Comment (RuoChengF) [reply pkg/web/src/modules/editClusterSlice.ts]: 已经去掉了

File: pkg/web/src/modules/editClusterSlice.ts (hunk 2 above)
Comment (zsnmwy) [reply pkg/web/src/modules/editClusterSlice.ts:26]: The range of 1-9.9 of the value is not correct.
The value will be passed to pql(Prometheus Query Language) string.

image

https://github.com/gocrane/crane/blob/087dc98230a5654abb556d5592f600983adc0f69/pkg/web/src/hooks/useCraneDiscount.ts#L4-L10

https://github.com/gocrane/crane/blob/087dc98230a5654abb556d5592f600983adc0f69/pkg/web/src/pages/Dashboard/Base/components/TopPanel.tsx#L10-L67

Comment (RuoChengF) [reply pkg/web/src/modules/editClusterSlice.ts:26]: 也就是取值范围 不能小于0 就可以了是吧 ,  还是说取值范围 在 0-100 之间

Comment (RuoChengF) [reply pkg/web/src/modules/editClusterSlice.ts]: 重新修复了一下  折扣不可为负数

**Comment by mfanjie** [comment]
@RuoChengF can you make the PR description as English, especially make the commit message as English.

**Comment by RuoChengF** [comment]
The Pr content submitted this time is to modify the discount input content shall not be lower than 0

File: pkg/web/src/pages/Settings/cluster/EditClusterModal.tsx [hunk 3]
```diff
@@ -338,8 +338,12 @@ export const EditClusterModal = React.memo(() => {
                       value={cluster.discount}
                       onChange={(value: InputValue) => {
                         let discount = value;
+                        let reg = /^\d+(\.{0,1}\d+){0,1}$/
```
Comment (qmhu) [inline pkg/web/src/pages/Settings/cluster/EditClusterModal.tsx:341]: Not a good way to implement it.

Comment (RuoChengF) [reply pkg/web/src/pages/Settings/cluster/EditClusterModal.tsx:341]: Of course, this is definitely not a good way to implement it. There are two main reasons why I write regular expressions directly inside components.
1. The web side has a low utilization rate for its own encapsulation method, and it seems to have multiple files, which is rather messy, which is one of the reasons why I put forward the improvement of "READE.MD" in the issue
2. My implementation idea is based on the current code style. If I repackage this thing into a method for import verification, the amount of code generated is too much.

**Pull request closed without merging**